code/modules/admin/admin_verbs.dm

```diff
@@ -1,3 +1,28 @@
+/datum/admins/proc/admin_sql()
+	set category = "Admin"
+	set name = "SQLQuery"
+
+	if(!check_rights(R_ADMIN))
+		to_chat(usr, "<span class='warning'>Not enough rights.</span>")
+		return
+	if(!CONFIG_GET(flag/sql_enabled))
+		to_chat(usr, "<span class='warning'>SQL disabled.</span>")
+		return
+	if(!SSdbcore.IsConnected())
+		to_chat(usr, "<span class='warning'>SQL not connected?!.</span>")
+		return
+	
+	var/rawQuery = input(usr, "What is your query", "Admin Query", "") as text|null
+	if(!findtext(rawQuery, regex("^select [\s*a-z,]+ from feedback(?:|\s+[a-z].*);$", "i")))
```

Comment:
It's essentially impossible to regex out user-input injections with regex, some examples with this regex:
`SELECT * from feedback WHERE 1=1; DROP TABLE feedback;`
`SELECT * from feedback;`
`SELECT * from feedback UNION ALL SELECT * FROM someothertable;`
`SELECT * from feedback WHERE 1=1 OR sleep(30000000000000000);`

https://regex101.com/r/sfsJGE/1

I'm not very good at sql injections so no doubt there'll be many more methods even with more restrictive parsing.